### PR TITLE
Replace test.sh with tox

### PR DESCRIPTION
**Pull request by MartinBasti**
# Maintainers will complete the following section

- [x] Commit messages are descriptive enough
- [x] Code coverage from testing does not decrease and new code is covered

### CODE REVIEW

**Comment by MartinBasti** [comment]
@tim-vk FYI

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,39 @@
+[tox]
```
Comment (ben-alkov) [inline tox.ini:1]: @MartinBasti; Have you seen https://www.seanh.cc/2018/11/14/tox/ ?

Comment (ben-alkov) [reply tox.ini:1]: FWIW, osbs-integration-tests' 'tox.ini' is structured like that :arrow_up:

Comment (MartinBasti) [reply tox.ini:1]: so do you want me to change it into that format?

**Review by rcerven — APPROVED** [review]

**Review by mkosiarc — APPROVED** [review]

**Review by ben-alkov — APPROVED** [review]

**Comment by MartinBasti** [comment]
release notes:
> - migrated tests to tox for easier testing and development

**Pull request merged**